http-ballerina/Ballerina.toml

```diff
@@ -1,147 +1,194 @@
-    version = "@netty.version@"
-    path = "./lib/netty-handler-proxy-@netty.version@.jar"
-    groupId = "io.netty"
-    modules = ["http"]
-
-    [[platform.libraries]]
-    artifactId = "netty-codec-http2"
-    version = "@netty.version@"
-    path = "./lib/netty-codec-http2-@netty.version@.jar"
-    groupId = "io.netty"
-    modules = ["http"]
-
-    [[platform.libraries]]
-    artifactId = "commons-pool"
-    version = "@commmon.pool.version@"
-    path = "./lib/commons-pool-@commmon.pool.version@.jar"
-    groupId = "commons-pool.wso2"
-    modules = ["http"]
-
-    [[platform.libraries]]
-    artifactId = "snakeyaml"
-    version = "@snakeyaml.version@"
-    path = "./lib/snakeyaml-@snakeyaml.version@.jar"
-    groupId = "org.wso2.orbit.org.yaml"
-    modules = ["http"]
-
-    [[platform.libraries]]
-    artifactId = "bcprov-jdk15on"
-    version = "@bouncycastle.version@"
-    path = "./lib/bcprov-jdk15on-@bouncycastle.version@.jar"
-    groupId = "org.bouncycastle"
-    modules = ["http"]
-
-    [[platform.libraries]]
-    artifactId = "bcpkix-jdk15on"
-    version = "@bouncycastle.version@"
-    path = "./lib/bcpkix-jdk15on-@bouncycastle.version@.jar"
-    groupId = "org.bouncycastle"
-    modules = ["http"]
-
-    [[platform.libraries]]
-    artifactId = "netty-tcnative-boringssl-static"
-    version = "@tcnative.version@"
-    path = "./lib/netty-tcnative-boringssl-static-@tcnative.version@.jar"
-    groupId = "io.netty"
-    modules = ["http"]
-
-    [[platform.libraries]]
-    artifactId = "mimepull"
-    version = "@mimepull.version@"
-    path = "./lib/mimepull-@mimepull.version@.jar"
-    groupId = "org.jvnet.mimepull"
-    modules = ["http"]
-
-    [[platform.libraries]]
-    artifactId = "org.testng"
-    version = "@testng.version@"
-    path = "./lib/testng-@testng.version@.jar"
-    groupId = "org.testng"
-    modules = ["http-tests"]
+[[dependency]]
+org = "ballerina"
+name = "io"
+version = "@stdlib.io.version@"
+
+[[dependency]]
+org = "ballerina"
+name = "cache"
+version = "@stdlib.cache.version@"
+
+[[dependency]]
+org = "ballerina"
+name = "file"
+version = "@stdlib.file.version@"
+
+[[dependency]]
+org = "ballerina"
+name = "auth"
+version = "@stdlib.auth.version@"
+
+[[dependency]]
+org = "ballerina"
+name = "crypto"
+version = "@stdlib.crypto.version@"
+
+[[dependency]]
+org = "ballerina"
+name = "mime"
+version = "@stdlib.mime.version@"
+
+[[dependency]]
+org = "ballerina"
+name = "time"
+version = "@stdlib.time.version@"
+
+[[dependency]]
+org = "ballerina"
+name = "runtime"
+version = "@stdlib.runtime.version@"
+
+[[dependency]]
+org = "ballerina"
+name = "log"
+version = "@stdlib.log.version@"
+
+[[dependency]]
+org = "ballerina"
+name = "system"
+version = "@stdlib.system.version@"
+
+[[dependency]]
+org = "ballerina"
+name = "reflect"
+version = "@stdlib.reflect.version@"
+
+[[dependency]]
+org = "ballerina"
+name = "stringutils"
+version = "@stdlib.stringutils.version@"
+
+[[dependency]]
```

Comment:
We can remove transitive dependencies, right? Is task directly used inside the HTTP package?

Reply:
Fixed via https://github.com/ballerina-platform/module-ballerina-http/commit/7ec4a4f867211c1b16ebc12f383ba1844ffff32c